lightning/src/ln/channel.rs

```diff
@@ -1751,31 +1751,59 @@ impl<Signer: Sign> Channel<Signer> {
 	}
 
 	/// Returns (inbound_htlc_count, htlc_inbound_value_msat)
-	fn get_inbound_pending_htlc_stats(&self) -> (u32, u64) {
+	fn get_inbound_pending_htlc_stats(&self) -> (u32, u64, u64, u64) {
```

Comment:
Method docs need updating. Does it make sense to start returning a `struct` here and for `get_outbound_pending_htlc_stats` so that we have named fields instead of a tuple?

Reply:
```
diff --git a/lightning/src/ln/channel.rs b/lightning/src/ln/channel.rs
index 87f4e6ea..022f65cf 100644
--- a/lightning/src/ln/channel.rs
+++ b/lightning/src/ln/channel.rs
@@ -273,6 +273,14 @@ enum HTLCInitiator {
        RemoteOffered,
 }
 
+/// An enum gathering stats on pending HTLCs, either inbound or outbound side.
+struct HTLCStats {
+       pending_htlcs: u32,
+       pending_htlcs_value_msat: u64,
+       on_counterparty_tx_dust_balance: u64,
+       on_holder_tx_dust_balance: u64,
+}
+
```